src/main/kotlin/com/ekino/oss/gradle/plugin/quality/QualityPlugin.kt

```diff
@@ -97,8 +92,9 @@ class QualityPlugin: Plugin<Project> {
     }
   }
 
-  private fun getTextResource(project: Project, file: String): TextResource {
-    return project.resources.text.fromString(QualityPlugin::class.java.getClassLoader().getResource(file).readText())
+  private fun getCheckstyleConfig(): File {
+    return QualityPlugin::class.java.classLoader?.getResource("checkstyle.xml")?.path
+            ?.let { File(it) }!!
```

Comment:
I think we should throw an exception with an accurate error message if the config file can't be found.

Reply:
Yes it's better with an exception with an explicit message.